web/gui/custom/README.md

```diff
@@ -129,104 +155,143 @@ If you need to set any other URL as the default Netdata server for all charts th
+settings though. Keep reading...
 
 ### The duration of the chart
 
-You can specify the duration of the chart (how much time of data it will show) using:
+You can specify the duration of the chart (how much time of data it will show)
+using:
 
 ```html
 <div data-netdata="unique.id"
-     data-after="AFTER_SECONDS"
-     data-before="BEFORE_SECONDS"
-     ></div>
+ data-after="AFTER_SECONDS"
+ data-before="BEFORE_SECONDS"
+ ></div>
 ```
 
-`AFTER_SECONDS` and `BEFORE_SECONDS` are numbers representing a time-frame in seconds.
+`AFTER_SECONDS` and `BEFORE_SECONDS` are numbers representing a time-frame in
+seconds.
 
 The can be either:
 
--   **absolute** unix timestamps (in javascript terms, they are `new Date().getTime() / 1000`. Using absolute timestamps you can have a chart showing always the same time-frame.
+-   **absolute** unix timestamps (in javascript terms, they are `new
+    Date().getTime() / 1000`. Using absolute timestamps you can have a chart
+    showing always the same time-frame.
 
--   **relative** number of seconds to now. To show the last 10 minutes of data, `AFTER_SECONDS` must be `-600` (relative to now) and `BEFORE_SECONDS` must be `0` (meaning: now). If you want the chart to auto-refresh the current values, you need to specify **relative** values.
+-   **relative** number of seconds to now. To show the last 10 minutes of data,
+    `AFTER_SECONDS` must be `-600` (relative to now) and `BEFORE_SECONDS` must
+    be `0` (meaning: now). If you want the chart to auto-refresh the current
+    values, you need to specify **relative** values.
 
 ### Chart sizes
 
 You can set the size of the chart using this:
 
 ```html
 <div data-netdata="unique.id"
-     data-width="WIDTH"
-     data-height="HEIGHT"
-     ></div>
+ data-width="WIDTH"
+ data-height="HEIGHT"
+ ></div>
 ```
 
-`WIDTH` and `HEIGHT` can be anything CSS accepts for width and height (e.g. percentages, pixels, etc).
-Keep in mind that for certain chart libraries, `dashboard.js` may apply an aspect ratio to these.
+`WIDTH` and `HEIGHT` can be anything CSS accepts for width and height (e.g.
+percentages, pixels, etc). Keep in mind that for certain chart libraries,
+`dashboard.js` may apply an aspect ratio to these.
 
-If you want `dashboard.js` to remember permanently (browser local storage) the dimensions of the chart (the user may resize it), you can add: `data-id="SETTINGS_ID"`, where `SETTINGS_ID` is anything that will be common for this chart across user sessions.
+If you want `dashboard.js` to remember permanently (browser local storage) the
+dimensions of the chart (the user may resize it), you can add: `data-id="
+SETTINGS_ID"`, where `SETTINGS_ID` is anything that will be common for this
+chart across user sessions.
 
 ### Netdata server
 
-Each chart can get data from a different Netdata server. You can give per chart the Netdata server using:
+Each chart can get data from a different Netdata server. You can give per chart
```

Comment:
`You can give per chart […]` → `You can specify the Netdata server to use for each chart using:`